src/test/java/com/microsoft/sqlserver/testframework/Constants.java

```diff
@@ -23,6 +23,7 @@ private Constants() {}
      * xAzureSQLDB - - - - For tests not compatible with Azure SQL Database
      * xAzureSQLDW - - - - For tests not compatible with Azure Data Warehouse
      * xAzureSQLMI - - - - For tests not compatible with Azure SQL Managed Instance
+     * xUnix - - - - - - - For tests not compatible with Unix platforms

```

Comment:
actually we don't need this, search for tests with this line:
        org.junit.Assume.assumeTrue(isWindows);